OpenRA.Mods.D2k/Activities/SwallowActor.cs

```diff
@@ -91,11 +95,13 @@ bool AttackTargets(Actor self, IEnumerable<Actor> targets)
 				}
 			});
 
+			var barrel = armament.CheckFire(self, facing, target);
```

Comment:
Yes, the warhead is still used for targeting. So IMO it only needs the damage value removed (and maybe Falloff changed to 0,0).